grapple/actions.py

```diff
@@ -227,6 +229,17 @@ def get_nested_field(cls, extract_key):
         if callable(cls_field):
             return cls_field(info, **kwargs)
 
+        # Expand HTML if the value's field is richtext
```

Comment:
Would this affect existing installations?

If so, is there any way we can make this opt-in, so that sites won't break on upgrade?

Reply:
This is on my list to review on Friday. it will most certainly affect existing installations

https://github.com/torchbox/wagtail-grapple/issues/204#issuecomment-968283281 mentions

> I think the better option woul be to add a setting that controls the output on both `RichTextField` and `RichTextBlock`